src/tests/cssNamespace.test.js

```diff
@@ -44,6 +44,17 @@ pluginTester({
         __dirname,
         './fixtures/nested_helper_styled_component.js'
       )
+    },
+    {
+      title: 'Do not namespace keyframes',
```

Comment:
Can we keep this consistent with the others?
`'does not add namespace to keyframes'`